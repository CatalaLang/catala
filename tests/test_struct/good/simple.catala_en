## Article

```catala
declaration structure S:
  data x content integer
  data y content integer

declaration scope A:
  context s content S
  context z content integer

scope A:
  definition s equals S {
    -- x : 1
    -- y : 2
  }
  definition z equals s.x + s.y
```
