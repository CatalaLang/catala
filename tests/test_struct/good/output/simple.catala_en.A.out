[RESULT] Computation successful! Results:
[RESULT] s = S {"x": 1, "y": 2}
[RESULT] z = 3
